.appveyor.yml

```diff
@@ -52,8 +52,9 @@ test_script:
 #- '%PYTHON%\%EXECUTABLE% test-installed.py -v -s %TEST_OPTIONS%' TODO TEST_OPTIONS with pytest?
 
 after_test:
-- python -m pip install codecov
-- codecov --file coverage.xml --name %PYTHON% --flags AppVeyor
+- curl -Os https://uploader.codecov.io/latest/windows/codecov.exe
+- chmod +x codecov
```

Comment:
```suggestion
```
I find this isn't necessary - https://github.com/radarhere/Pillow/commit/4866fc4443f1e7e9b30841cd4f56bc08199ebd1e / https://ci.appveyor.com/project/radarhere/pillow/builds/46769617/job/j08ajcwcui0e2bab